prow/README.md

```diff
@@ -52,6 +52,7 @@ Its structure looks as follows:
   └── plugins.yaml          # The file with Prow plugins configuration
   └── create-secrets-for-workload-cluster.sh  # The script which creates a Secret in the Prow cluster, used to access the workload cluster
   └── set-up-workload-cluster.sh  # The script which prepares the workload cluster to be used by the Prow cluster
+  └── static-files          # Files that will be uploaded to the nginx web server. To configure how traffic is routed to these files, update the prow ingress paths and the web server configuration ConfigMap.
```

Comment:
This is not the right place to provide details how to work with this files. Please update prow/cluster/README.md with such details. Place some info about how content of static-files is updated.

Reply:
@dekiel the folder is at `prow/static-files` not `prow/cluster/static-files`

Reply:
I know, but still you put information about how to work with files in place where we have short description what a directory contains. prow/cluster/README.md is my proposal where to place some details about how to use and work with these files. I would even vote for moving static-files somewhere under prow/cluster directory as these files are used on a prow cluster only. They are not a standalone tool and have not more usage than prow cluster.